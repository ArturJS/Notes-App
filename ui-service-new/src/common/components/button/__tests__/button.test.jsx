import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../button.jsx';

describe('<Button />', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(
                <Button type="submit" theme="hot" className="btn-test-class">
                    <span>Inner content...</span>
                </Button>
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
