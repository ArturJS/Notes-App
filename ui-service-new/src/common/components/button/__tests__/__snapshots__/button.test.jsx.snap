// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> renders correctly 1`] = `
<button
  className="btn btn-gradient-basis btn-hot btn-test-class"
  onClick={[Function]}
  type="submit"
>
  <span
    className="btn-content"
  >
    <span>
      Inner content...
    </span>
  </span>
</button>
`;
